src/logging.ts

```diff
@@ -113,13 +113,17 @@ export const logSubscriptionOperation = <TLogger extends Logger = Logger>({
   result,
   message,
   logLevel,
+  resolveCustomLogger,
 }: {
   args: ExecutionArgs
   result?: ExecutionResult
   message?: string
   logLevel?: keyof LoggerLogFunctions<TLogger>
+  resolveCustomLogger?: (context: GraphQLContext) => TLogger
```

Comment:
Minor naming thing, is the 'custom' part of this name meaningful?
```suggestion
  resolveLogger?: (context: GraphQLContext) => TLogger
```